docs/en/table_design/indexes/Bitmap_index.md

```diff
@@ -2,19 +2,26 @@
 displayed_sidebar: "English"
 ---
 
-# Bitmap indexing
+# Bitmap indexes
 
 This topic describes how to create and manage a bitmap index, along with usage cases.
 
 A bitmap index is a special database index that uses bitmaps, which are an array of bits. A bit is always in one of two values: 0 and 1. Each bit in the bitmap corresponds to a single row in the table. The value of each bit depends upon the value of the corresponding row.
 
-A bitmap index can help improve the query performance on a given column. If a query hits a sort key column, StarRocks efficiently returns the query result by using the [prefix index](../table_design/Sort_key.md). However, the prefix index entry for a data block cannot exceed 36 bytes in length. If you want to improve the query performance on a column, which is not used as a sort key, you can create a bitmap index for the column.
+A bitmap index can help improve the query performance on a given column. If a query hits a sort key column, StarRocks efficiently returns the query result by using the [prefix index](./indexes_overview.md#prefix-indexes). However, the prefix index entry for a data block cannot exceed 36 bytes in length. If you want to improve the query performance on a column, which is not used as a sort key, you can create a bitmap index for the column.
+
+Bitmap indexes are generally suitable for columns with high cardinality. Bitmap indexes are a good choice when  Bitmap indexes can exhibit high selectivity, and its filtering effect (number of data rows filtered by the Bitmap index/total number of data rows) is lower than one in ten thousand.
+
+To evaluate the acceleration performance of Bitmap indexes in StarRocks, queries are performed on a 100 GB SSB dataset in StarRocks. The test results are as follows:
+
+-  when bitmap indexes are added on high-cardinality columns (in this dataset, a noticeable performance improvement can be observed when the cardinality reaches the magnitude of 100,000).
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/18c10cbe-0d14-42fb-b9ad-da715a8fee34)
extra space, and "when" should be changed to "When"